core/exception_vectors.S

```diff
@@ -81,7 +83,9 @@ KernelExceptionVector:
         .type   KernelExceptionVector, @function
 
         break   1, 0
-        call0   sdk_user_fatal_exception_handler
+        mov     a2, a1
+        movi    a3, 0
```

Comment:
I haven't had a chance to check this, but doesn't this clobber the values of a2 and a3 before we have a chance to print them to the user?